src/main/java/com/tenx/ms/retail/order/domain/OrderElementEntity.java

```diff
@@ -0,0 +1,34 @@
+package com.tenx.ms.retail.order.domain;
+
+import com.tenx.ms.retail.product.domain.ProductEntity;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+
+import javax.persistence.*;
+
+@Entity
+@Data
+@Table(name="retail_order_element")
+public class OrderElementEntity implements Cloneable {
```

Comment:
Now I see that there is an OrderEntity.
This one is for "OrderItemEntity" or "OrderProductEntity".
I find "element" too general. But it's your call.